client/src/App.js

```diff
@@ -1,45 +1,39 @@
-import React from 'react';
+import React, {Component} from 'react';
 import SelectedFoods from './SelectedFoods';
 import FoodSearch from './FoodSearch';
 
-const App = React.createClass({
-  getInitialState: function () {
-    return {
-      selectedFoods: [],
-    };
-  },
-  render: function () {
+class App extends Component {
+  state = {
```

Comment:
Yeah, I mostly just commented on the title being "use ES6 syntax" when this piece indeed was _not_ ES6 syntax. I might be just splitting hairs, though :smile: